File: Java/Loop/Smallest Divisible Digit Product I.java
class Solution {
    int check(int n){
        int pro=1;
        while(n!=0){
            pro*=n%10;
            n/=10;
        }
        return pro;
    }
    public int smallestNumber(int n, int t) {
        while(check(n)%t!=0){
            n++;
        }
        return n;
    }
}

File: Python/Loop/Smallest Divisible Digit Product I.py
class Solution:
    def check(self, n:int)->int:
        pro=1
        while n!=0:
            pro*=n%10
            n=n//10
        return pro

    def smallestNumber(self, n: int, t: int) -> int:
        while self.check(n)%t!=0:
            n+=1
        return n
        
